unittests/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/IFDSUninitializedVariablesTest.cpp

```diff
@@ -49,6 +49,7 @@ class IFDSUninitializedVariablesTest : public ::testing::Test {
   void TearDown() override {
     delete IRDB;
     delete TH;
+    delete PT;
     delete ICFG;
     delete UninitProblem;
```

Comment:
The same could be applied here, regarding `unique_ptr` instead of `new` (see bellow).
Would be nice if you could quickly refactored the tests here and replace all the `new` calls with `unique_ptrs`, so we can prevent this kind of bug in the future. :+1:

Reply:
Updated. 

All the places using raw pointers are updated using unique_ptr.